src/components/uploadsGalleryModal/container/uploadsGalleryModal.tsx

```
@@ -259,9 +260,11 @@ export const UploadsGalleryModal = forwardRef(
         }
 
         // filter out oversized images (server limit is 30MB)
-        const oversized = media.filter((item) => item.size && item.size > MAX_IMAGE_UPLOAD_SIZE);
+        const oversized = media.filter(
+          (item) => item && item.size && item.size > MAX_IMAGE_UPLOAD_SIZE,
+        );
         if (oversized.length > 0) {
-          media = media.filter((item) => !item.size || item.size <= MAX_IMAGE_UPLOAD_SIZE);
+          media = media.filter((item) => item && (!item.size || item.size <= MAX_IMAGE_UPLOAD_SIZE));
```

ESLint

[error] Replace `(item)·=>·item·&&·(!item.size·||·item.size·<=·MAX_IMAGE_UPLOAD_SIZE)` with `⏎············(item)·=>·item·&&·(!item.size·||·item.size·<=·MAX_IMAGE_UPLOAD_SIZE),⏎··········`

(prettier/prettier)